beetsplug/info.py

```diff
@@ -239,4 +239,5 @@ def filter_(data):
 
 
 def identity(val):
+    val.pop('path', None) # path is fetched from item
```

Comment:
[We require two spaces before every inline comment](https://travis-ci.org/beetbox/beets/jobs/460820041#L1087);

```suggestion
    val.pop('path', None)  # path is fetched from item
```